sdk/mainnet_consts.go

```diff
@@ -189,4 +190,5 @@ var KnownAutomaticRelayerEmitters = []struct {
 	{ChainId: vaa.ChainIDBlast, Addr: "00000000000000000000000027428DD2d3DD32A4D7f7C497eAaa23130d894911"},
 	{ChainId: vaa.ChainIDMantle, Addr: "00000000000000000000000027428dd2d3dd32a4d7f7c497eaaa23130d894911"},
 	{ChainId: vaa.ChainIDXLayer, Addr: "00000000000000000000000027428dd2d3dd32a4d7f7c497eaaa23130d894911"},
+	{ChainId: vaa.ChainIDSnaxchain, Addr: "00000000000000000000000027428DD2d3DD32A4D7f7C497eAaa23130d894911"},
```

Comment:
I'm sure this is fine, but since the contract source code isn't verified is there another way for us to verify this is indeed the correct address?

Reply:
@solanoepalacio could you please verify the relayer contract?

Reply:
This contract has since been [verified on the explorer](https://explorer.snaxchain.io/address/0x27428DD2d3DD32A4D7f7C497eAaa23130d894911?tab=contract_code) along with its [implementation](https://explorer.snaxchain.io/address/0x7A0a53847776f7e94Cc35742971aCb2217b0Db81?tab=contract)

Reply:
Thanks!